postgresql-ballerina/Package.md

```diff
@@ -0,0 +1,68 @@
+## Package overview
+
+This Package provides the functionality required to access and manipulate data stored in a PostgreSQL database.  
+
+**Prerequisite:** Add the PostgreSQL driver JAR as a native library dependency in your Ballerina project. 
+This Package uses database properties from the PostgreSQL version 42.2.18. Therefore, it is recommended to use a 
+PostgreSQL driver version 42.2.18. Then, once you build the project by executing the `ballerina build`
+command, you should be able to run the resultant by executing the `ballerina run` command.
+
+E.g., The `Ballerina.toml` content.
+Change the path to the JDBC driver appropriately.
+
+```toml
+[package]
+org = "sample"
+name = "postgresql"
+version= "0.1.0"
+
+[[platform.java11.dependency]]
+artafactId = "postgresql-connector-java"
```

Comment:
fix the typo

Reply:
I will fix

Reply:
Fixed. Commit id is c6c5bfc78b2774c6bc276f5857b7e2b9e98d7bf7